src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.csproj

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```

Comment:
The System.Data.SqlClient csproj files do not have any OS-Specific includes so the Debug default will be fine. The Runtime.PInvoke csproj also does not have any OS-Specific includes but also has a default include in a later PropertyGroup, so the Windows Specific one was actually incorrect